waiter/src/waiter/main.clj

```diff
@@ -77,7 +77,8 @@
    :state core/state
    :http-server (pc/fnk [[:curator kv-store]
                          [:routines generate-log-url-fn waiter-request?-fn websocket-request-acceptor]
```

Comment:
Nitpick: Unnecessary change for this PR.

Reply:
🤷‍♂ fixed